packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -1106,4 +1108,14 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
       </Label>
     ) : null;
   };
+
+  /**
+   * Returns true if dropdown should set to open on focus.
+   * Otherwise, isOpen state should be toggled on click
+   */
+  private _shouldOpenOnFocus(): boolean {
+    const { hasFocus } = this.state;
+    const { openOnKeyboardFocus } = this.props;
+    return !this._isFocusedByClick && openOnKeyboardFocus === true && !hasFocus;
```

Comment:
Is there a reason we cannot do?

```suggestion
    return !this._isFocusedByClick && openOnKeyboardFocus && !hasFocus;
```

Reply:
i missed your comment and the PR was auto-merged. sorry about that
To answer you. reason is `openOnKeyboardFocus` is type `boolean | undefined`
```
true && undefined // undefined
true && undefined === true // false
```